libraries/types/dag_block/include/dag/dag_block.hpp

```diff
@@ -93,8 +93,15 @@ class DagBlock {
   void verifyVdf(const SortitionParams &vdf_config, const h256 &proposal_period_hash) const;
   bytes rlp(bool include_sig) const;
 
+  /**
+   * @brief Returns dag block data rlp stream
+   *
+   * @param include_sig
+   * @return dev::RLPStream
+   */
+  dev::RLPStream streamRLP(bool include_sig) const;
```

Comment:
include_sig = false by default?

Reply:
or in rlp?

Reply:
any reason ? It was without default value until now and I dont see why it should be